### PR TITLE
add local editor

### PR DESCRIPTION
**Pull request by jessitron**
paired with @ddgenome on this one

It supports a blog post I'm about to publish.

### CODE REVIEW

**Comment by jessitron** [comment]
there are some casing changes that will be completed by one of David's PRs

**Comment by jessitron** [comment]
The local `rug install` and `rug test` are working differently than the one on Travis. I think the error in the build is legit (because of the rug npm module upgrade, that editor prolly needs updated) but why don't I see the error locally??

**Comment by jessitron** [comment]
'k so this is why we haven't set the package.json to 0.13.0 yet

**Comment by jessitron** [comment]
so @ddgenome we could merge this and set the package.json forward separately?

**Comment by jessitron** [comment]
ah got it! I think clearing the Travis cache for the master branch (and this PR) did the trick

**Comment by jessitron** [comment]
either that or something else someone did

**Comment by jessitron** [comment]
OK now this is REALLY ready to merge

File: README.md [hunk 1]
```diff
@@ -390,6 +390,33 @@ will create a directory named `ruggery` and populate it with a working
 Rug archive project with a standard layout and TypeScript dependencies
 and a simple editor.  You can use the editors in this project to add
 more Rugs to the generated project.
+### AddLocalEditor
```
Comment (ddgenome) [inline README.md]: Can you add a space above? Better yet, put it in alphabetical order in the README?

File: .atomist/tests/AddLocalEditor.rt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+scenario AddLocalEditor is added to your project by AddAddLocalEditor
```
Comment (ddgenome) [inline .atomist/tests/AddLocalEditor.rt]: This is totally useless: DSL & not accurate.

**Comment by jessitron** [comment]
To make tests I'd add them to the editor, and I don't want to do that as part of this PR. That's a separate problem.

We can open an issue and I can add it later?
I really would like to publish this post.

**Pull request merged**